infra/Pos/Diffusion/Types.hs

```diff
@@ -24,15 +25,15 @@ data Diffusion m = Diffusion
       getBlocks          :: NodeId
                          -> BlockHeader
                          -> [HeaderHash]
-                         -> m (Either GetBlocksError [Block])
+                         -> m [Block]
       -- This is needed because there's a security worker which will request
```

Comment:
Why not make these comments proper haddock comments btw?

Reply:
I'll do that